src/components/loader-with-overlay.js

```diff
@@ -33,14 +31,7 @@ const LoaderWithOverlay = ({
     return (
         <div className={classes.overlay} id={color}>
             <CircularProgress color={color} size={loaderSize} />
-            <div
-                className={classes.message}
-                style={{
-                    fontSize: loadingMessageSize,
-                }}
-            >
-                <FormattedMessage id={loadingMessageText} />
-            </div>
+            <FormattedMessage id={loadingMessageText} />
```

Comment:
Why did you delete all this? You are sure that this modification will not impact the loader displayed before the map is loaded?

Reply:
I just remove the div element to have a column type alignment for the circular progress element and the message element